src/ui/atoms/card/card.scss

```diff
@@ -19,37 +19,36 @@ $sizes-map: (
 
 .okp4-card-main {
   @include with-theme() {
-    clip-path: polygon(30px 0, 100% 0, 100% 100%, 0 100%, 0 30px);
-    padding: 1px;
-
-    &.border {
-      background: themed(border);
-    }
-
-    .okp4-card-container {
-      @each $color-key, $color-value in $colors {
-        display: flex;
-        flex-direction: column;
-        justify-content: space-between;
-        gap: 10px;
-        box-sizing: border-box;
+    @each $size in $sizes {
+      &.#{$size} {
         clip-path: polygon(30px 0, 100% 0, 100% 100%, 0 100%, 0 30px);
-        max-height: map-get($sizes-map, large-height) - 2px;
-        max-width: map-get($sizes-map, large-width) - 2px;
-        overflow: auto;
-        word-break: break-word;
+        padding: 1px;
+        width: 100%;
+        max-width: map-get($sizes-map, #{$size}-width) - 2px;
```

Comment:
I am not sure that we need this `- 2px` as we discussed yesterday. 
What do you think @fredericvilcot, is there a specific reason?

Reply:
You're right I think we can remove all `- 2px`, I don't see how it will change the component other than giving it the `max-width` set by the map

Reply:
💯 agree